inc/document_item.class.php

```diff
@@ -85,14 +85,26 @@ function canCreateItem() {
 
    function prepareInputForAdd($input) {
 
+      if (empty($input['itemtype']) || trim($input['itemtype']) == '') {
```

Comment:
`!class_exists($input['itemtype'])` could be a good test too.

Reply:
Good catch, I've added this one

Reply:
It will fail if class is surrounded by spaces, so should you remove the `trim` check. I think we should not support usage of itemtype that does not exactly matches classname.

Reply:
Done as well